docs/source/scalarstats.rst

```diff
@@ -158,6 +158,10 @@ Quantile and Friends
     w = rand(n)
     xk = median(x, weights(w))
 
+.. function:: wquantile(x, w, q)
```

Comment:
Arguments are not in the same order as in the code. Also, better rename `q` to `p` to follow Base.